src/ser-ext-ondemand.ts

```diff
@@ -355,13 +344,9 @@ export = {
         // empty function to avoid braking when resize method is required
     },
     controller: ["$scope", function (scope: IVMScopeExtended) {
-
-        //#region Logger
-        logging.LogConfig.SetLogLevel("*", scope.layout.properties.loglevel);
-        let logger = new logging.Logger("Main");
-        //#endregion
+        let logger = new LoggerSing(scope.layout.properties.loglevel);
```

Comment:
why not use LoggerSing.getInstance() ? so why a LoggerSingleton is required ?

Reply:
you are absolutly right, this should be a factory, will refactor it. Thanks

Reply:
removed Singolton, fixed with https://github.com/senseexcel/ser-ext-ondemand/commit/39ff29d840b6de7135be57615e679bf72f18be35